core/controllers/android_e2e_config.py

```diff
@@ -130,7 +130,7 @@ def post(self):
         # Add other structures to the topic.
         topic.add_canonical_story(story_id)
         topic.add_uncategorized_skill_id(skill_id)
-        topic.add_subtopic(1, 'Test Subtopic Title')
+        topic.add_subtopic(1, 'Test Subtopic Title', 'url-frag')
```

Comment:
```suggestion
        topic.add_subtopic(1, 'Test Subtopic Title', 'testsubtop')
```

ditto as in admin.py